core/primitives/src/sharding.rs

```diff
@@ -81,7 +81,7 @@ pub struct ShardChunkHeaderInner {
     pub encoded_length: u64,
     pub height_created: BlockHeight,
     /// Shard index.
```

Comment:
Can you also extend this documentation to make it clear why we have `shard_ord`, and not `shard_id`?